recipes/lmdb/all/CMakeLists.txt

```
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+# LMDB library:
+
+add_library(lmdb
+    source_subfolder/lmdb.h
+    source_subfolder/mdb.c
+    source_subfolder/midl.c
+    source_subfolder/midl.h
+)
+
+set_target_properties(lmdb PROPERTIES PUBLIC_HEADER
+    source_subfolder/lmdb.h
+)
+
+if(NOT WIN32)
```

I'm not sure, but I don't think pthread is needed on Macos.
```suggestion
if(UNIX AND NOT APPLE)
```

---

I think it's required everywhere but Linux.